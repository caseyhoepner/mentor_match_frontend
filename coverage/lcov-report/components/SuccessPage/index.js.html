<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for components/SuccessPage/index.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../index.html">All files</a> / <a href="index.html">components/SuccessPage</a> index.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>7/7</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">75% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>3/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>7/7</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">3x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import { Link, withRouter } from 'react-router-dom';
import './SuccessPage.css';
&nbsp;
export const SuccessPage = (props) =&gt; {
  const { state } = props.history.location;
  let successMessage;
&nbsp;
  if (state.from === "/new-student-form") {
    successMessage = (
      &lt;div className='sp-text'&gt;
        &lt;p&gt;New student successfully added!&lt;/p&gt;
        &lt;Link
          className='sp-new-student-btn'
          to='/new-student-form'
        &gt;+ Add Another Student
        &lt;/Link&gt;
      &lt;/div&gt;
      )
  } else <span class="missing-if-branch" title="else path not taken" >E</span>if (state.from === "/new-mentor-form") {
    successMessage = (
      &lt;div className='sp-text'&gt;
        &lt;p&gt;Thank you for your interest in mentoring our students. We will reach out to you shortly.&lt;/p&gt;
        &lt;p&gt;If you have any questions, reach out to 
          &lt;a 
            className='sp-email-link'
            href="mailto:kayt@turing.io"&gt; kayt@turing.io
          &lt;/a&gt;.&lt;/p&gt;
      &lt;/div&gt;
    )
  }
&nbsp;
&nbsp;
  return (
    &lt;div className='sp-container'&gt;
      &lt;img 
        className='sp-icon'
        src={require('../../utils/assets/success-check.svg')}
        alt='Success Icon'
        /&gt;
      &lt;h1 className='sp-title'&gt;Success!&lt;/h1&gt;
      { successMessage }
    &lt;/div&gt;
  )
}
&nbsp;
export default withRouter(SuccessPage);</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sun Jan 13 2019 20:40:28 GMT-0700 (MST)
</div>
</div>
<script src="../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../sorter.js"></script>
</body>
</html>
