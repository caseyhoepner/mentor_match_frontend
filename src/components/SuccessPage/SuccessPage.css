.sp-container {
  margin: auto;
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 100px auto;
  width: 50%;
}

.sp-title {
  font-family: 'Archivo Black', sans-serif;
  font-weight: normal;
  font-size: 5rem;
  margin: 20px 0;
}

.sp-text {
  padding: 0px 50px;
  font-size: 1.5rem;
  display: flex;
  flex-direction: column;
  align-items: center;
}

.sp-new-student-btn {
  background-color: #10f1fc;
  color: #fff;
  padding: 10px 15px;
  border-radius: 50px;
  margin: 100px 0;
}

.sp-new-student-btn:hover {
  background-color: #0CC0C9;
}

.sp-icon {
  width: 100px;
  height: 100px;
}

.sp-email-link {
  color: #10f1fc;
}

.sp-email-link:hover {
  color: #0CC0C9;
}