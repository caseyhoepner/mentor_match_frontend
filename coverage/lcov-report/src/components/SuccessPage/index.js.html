<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components/SuccessPage/index.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/components/SuccessPage</a> index.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">14.29% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>1/7</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">14.29% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>1/7</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import { Link, withRouter } from 'react-router-dom';
import './SuccessPage.css';
&nbsp;
export const SuccessPage = <span class="fstat-no" title="function not covered" >(p</span>rops) =&gt; {
  // const { history } = props;
  const { state } = <span class="cstat-no" title="statement not covered" >props.history.location;</span>
  let successMessage;
&nbsp;
<span class="cstat-no" title="statement not covered" >  if (state.from === "/new-student-form") {</span>
<span class="cstat-no" title="statement not covered" >    successMessage = (</span>
      &lt;div&gt;
        &lt;p&gt;New student successfully added!&lt;/p&gt;
        &lt;Link
          className='sp-new-student-btn'
          to='/new-student-form'
        &gt;Add Another Student
        &lt;/Link&gt;
      &lt;/div&gt;
      )
  } else <span class="cstat-no" title="statement not covered" >if (state.from === "/new-mentor-form") {</span>
<span class="cstat-no" title="statement not covered" >    successMessage = (</span>
      &lt;div&gt;
        &lt;p&gt;Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.&lt;/p&gt;
      &lt;/div&gt;
    )
  }
&nbsp;
&nbsp;
<span class="cstat-no" title="statement not covered" >  return (</span>
    &lt;div className='sp-container'&gt;
      &lt;h1&gt;Success!&lt;/h1&gt;
      { successMessage }
    &lt;/div&gt;
  )
}
&nbsp;
export default withRouter(SuccessPage);</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Wed Jan 02 2019 11:10:42 GMT-0700 (MST)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
