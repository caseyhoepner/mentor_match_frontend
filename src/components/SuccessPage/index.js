import React from 'react';
import { Link, withRouter } from 'react-router-dom';
import './SuccessPage.css';

export const SuccessPage = (props) => {
  // const { history } = props;
  const { state } = props.history.location;
  let successMessage;

  if (state.from === "/new-student-form") {
    successMessage = (
      <div>
        <p>New student successfully added!</p>
        <Link
          className='sp-new-student-btn'
          to='/new-student-form'
        >Add Another Student
        </Link>
      </div>
      )
  } else if (state.from === "/new-mentor-form") {
    successMessage = (
      <div>
        <p>Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.</p>
      </div>
    )
  }


  return (
    <div className='sp-container'>
      <h1>Success!</h1>
      { successMessage }
    </div>
  )
}

export default withRouter(SuccessPage);