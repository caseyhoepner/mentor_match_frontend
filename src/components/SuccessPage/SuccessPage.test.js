import React from 'react';
import { shallow } from 'enzyme';
import SuccessPage from './';

describe ('SuccessPage', () => {
  let wrapper;
  let mockHistory = {
    location: {
      state: {
        from: '/new-student-form'
      }
    }
  }

  beforeEach(() => {
    wrapper = shallow(<SuccessPage history={mockHistory} />)
  })

  it('should match the snapshot when it came from NewStudentForm', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should match the snapshot when it came from NewMentorForm', () => {
    let mockHistory2 = {
      location: {
        state: {
          from: '/new-mentor-form'
        }
      }
    }
    wrapper = shallow(<SuccessPage history={mockHistory2} />)
    expect(wrapper).toMatchSnapshot()
  });
})