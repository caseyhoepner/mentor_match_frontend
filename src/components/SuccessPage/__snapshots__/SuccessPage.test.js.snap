// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessPage should match the snapshot when it came from NewMentorForm 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SuccessPage
    history={
      Object {
        "location": Object {
          "state": Object {
            "from": "/new-mentor-form",
          },
        },
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Success!
        </h1>,
        <div>
          <p>
            Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.
          </p>
        </div>,
      ],
      "className": "sp-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Success!",
        },
        "ref": null,
        "rendered": "Success!",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <p>
            Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.
          </p>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.",
          },
          "ref": null,
          "rendered": "Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.",
          "type": "p",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Success!
          </h1>,
          <div>
            <p>
              Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.
            </p>
          </div>,
        ],
        "className": "sp-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Success!",
          },
          "ref": null,
          "rendered": "Success!",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <p>
              Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.
            </p>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.",
            },
            "ref": null,
            "rendered": "Thank you for signing up to be a mentor! We'll be in touch when we find you a student to mentor.",
            "type": "p",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;

exports[`SuccessPage should match the snapshot when it came from NewStudentForm 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SuccessPage
    history={
      Object {
        "location": Object {
          "state": Object {
            "from": "/new-student-form",
          },
        },
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Success!
        </h1>,
        <div>
          <p>
            New student successfully added!
          </p>
          <Link
            className="sp-new-student-btn"
            replace={false}
            to="/new-student-form"
          >
            Add Another Student
          </Link>
        </div>,
      ],
      "className": "sp-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Success!",
        },
        "ref": null,
        "rendered": "Success!",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              New student successfully added!
            </p>,
            <Link
              className="sp-new-student-btn"
              replace={false}
              to="/new-student-form"
            >
              Add Another Student
            </Link>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "New student successfully added!",
            },
            "ref": null,
            "rendered": "New student successfully added!",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Add Another Student",
              "className": "sp-new-student-btn",
              "replace": false,
              "to": "/new-student-form",
            },
            "ref": null,
            "rendered": "Add Another Student",
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Success!
          </h1>,
          <div>
            <p>
              New student successfully added!
            </p>
            <Link
              className="sp-new-student-btn"
              replace={false}
              to="/new-student-form"
            >
              Add Another Student
            </Link>
          </div>,
        ],
        "className": "sp-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Success!",
          },
          "ref": null,
          "rendered": "Success!",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                New student successfully added!
              </p>,
              <Link
                className="sp-new-student-btn"
                replace={false}
                to="/new-student-form"
              >
                Add Another Student
              </Link>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "New student successfully added!",
              },
              "ref": null,
              "rendered": "New student successfully added!",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Add Another Student",
                "className": "sp-new-student-btn",
                "replace": false,
                "to": "/new-student-form",
              },
              "ref": null,
              "rendered": "Add Another Student",
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
